libs/utils/decorate.js

```diff
@@ -260,3 +260,21 @@ export function applyInViewPortPlay(video) {
     observer.observe(video);
   }
 }
+
+export function turnAnchorIntoVideo({ hash, src, anchorTag }) {
+  const { dataset, parentElement } = anchorTag;
+  const attrs = getVideoAttrs(hash, dataset);
+  const video = `<video ${attrs}></video>`;
+  anchorTag.insertAdjacentHTML('afterend', video);
+  const videoEl = parentElement.querySelector('video');
+  createIntersectionObserver({
+    el: parentElement,
+    options: { rootMargin: '1000px' },
```

Comment:
Just one last curiosity: I know this value was here to begin with, but are we sure it's the right one for our performance needs? Could there be situations where we prematurely load videos? Could the value be lower?

Reply:
If you have a better suggestion, I'm happy to adapt it 😁 It was initially introduced in [this PR](https://github.com/adobecom/milo/pull/1343) by @seanchoi-dev 

I don't see any context // investigation on how that value got to be. Should we use sth similar to the footer -> 300px?